### ISSUE TITLE
Enhancing security of data submitted to the TSA

### ISSUE THREAD

**Issue by thomwithoutanh** [the-engine-room/alidade#8]
Currently, the information that users save in the TSA is saved in the database as plaintext (I think). It would be ideal if this data could stored more securely (so we can't see it ourselves). 
They could be giving us (potentially sensitive) information about their organisations’ capacity and tool decisions. We don't need to see this, and to walk the 'responsible data' walk it would be great to prevent us from being able to do so.

Is this possible @IronRunes?

**Comment by IronRunes** [comment]
yes @thomwithoutanh , it is indeed possible. It is somewhat complex to pull off, because of how we would want to actually encrypt this data uniquely for each user, but it can be done (pretty much like we do for passwords, but on a per-user basis instead of an per-application-install approach).

**Comment by thomwithoutanh** [comment]
@IronRunes Ok, good to know. Suggestion: let's put it in Icebox for now, until we have a complete set of issues based on the UX testing. Then we can estimate how much time it would take and see if we have room to do it?

**Comment by thomwithoutanh** [comment]
Alan and I discussed this and suggested we put it aside. 

The main thing is how to communicate that the data we are collecting is being stored securely. @IronRunes I might need your help in getting the content to write in here. But this can come later.
